### ISSUE TITLE
Sanitise Data leaving Travis

### ISSUE THREAD

**Issue by matt-oc** [nodezoo/nodezoo-travis#20]

**Comment by mcdonnelldean** [comment]
@matt-oc  Can you run them through a pretty printer and use the code formatting, otherwise I can't read them

**Comment by matt-oc** [comment]
I like ID, Group, Last State and Time, they are valuable and seem to be consistent and likely to have correct information (the language for instance is usually incorrect on Travis)

**Comment by matt-oc** [comment]
@mcdonnelldean I'm trying to find one to give a nice output, bear with me!

**Comment by mcdonnelldean** [comment]
Also, A link to the latest build

**Comment by matt-oc** [comment]
Yes thats a good point so we have ID, Group, Last State and time, and a URL link to the Travis page.

That output looks a bit better now too :+1:

**Comment by mcdonnelldean** [comment]
Can you make a fake payload at the bottom of your original message, with the ones you think are right, I can edit accordingly then.

**Comment by mcdonnelldean** [comment]
As a JSON object

**Comment by matt-oc** [comment]
Initial proposed object now implemented

**Comment by mcdonnelldean** [comment]
@matt-oc Lets leave this and the others open for the time being, just in case we want more from them

**Comment by matt-oc** [comment]
@mcdonnelldean Yep will do, I'm updating the original comment whenever the data is changed so people know where we are with regards to data.
